lib/go/thrift/simple_server.go

```diff
@@ -54,10 +55,10 @@ var ServerConnectivityCheckInterval = time.Millisecond * 5
  * This will work if golang user implements a conn-pool like thing in client side.
  */
 type TSimpleServer struct {
-	closed int32
-	wg     sync.WaitGroup
-	mu     sync.Mutex
-
+	closed                 int32
+	wg                     sync.WaitGroup
+	mu                     sync.Mutex
+	stopChan               chan struct{}
```

Comment:
nit: I would still prefer to have a blank line after this line so we have the logical "groups" of the fields of a `TSimpleServer`.